actioncable/lib/action_cable/channel/test_case.rb

```diff
@@ -62,6 +62,25 @@ def initialize(identifiers = {})
       def transmit(cable_message)
         transmissions << cable_message.with_indifferent_access
       end
+
+      def connection_identifier
+        unless defined? @connection_identifier
+          @connection_identifier = connection_gid identifiers.filter_map { |id| instance_variable_get("@#{id}") }
+        end
+
+        @connection_identifier
+      end
+
+      private
+        def connection_gid(ids)
+          ids.map do |o|
+            if o.respond_to? :to_gid_param
+              o.to_gid_param
+            else
+              o.to_s
+            end
+          end.sort.join(":")
+        end
```

Comment:
As this is copying mostly the following code
https://github.com/rails/rails/blob/75ac626c4e21129d8296d4206a1960563cc3d4aa/actioncable/lib/action_cable/connection/identification.rb#L27-L44

...would it make sense to include the whole module instead?
```
   include ActionCable::Identification::Identification
```

Reply:
I honestly don't know!

I am not ultra-familiar with this part of the codebase, and what I added/copied was the minimum required to make `ConnectionStub` live up to its name.

What happened when you tried it?

Reply:
I haven't tried it 😄 and I'm not familiar with this part either.
Looking at the implementation I noticed the similarity and wondered if we could reuse it.

Reply:
Just eyeballing them both, it seems like there could be some awkwardness with how the `ActionCable::Connection::Identification` concern is intended for devs to declare identifiers in their `Connection` class with `identified_by`, whereas the `stub_connection` method accepts identifiers via parameter.

So... close but not quite? would be my call. Happy to be proven wrong - I just want the end result.